Source/WebCore/rendering/svg/RenderSVGResourceClipper.cpp

```diff
@@ -211,27 +233,25 @@ bool RenderSVGResourceClipper::hitTestClipContent(const FloatRect& objectBoundin
     return layer()->hitTest(hitType, result);
 }
 
-FloatRect RenderSVGResourceClipper::resourceBoundingBox(const RenderObject& object, RepaintRectCalculation)
+FloatRect RenderSVGResourceClipper::resourceBoundingBox(const RenderObject& object, RepaintRectCalculation repaintRectCalculation)
 {
     FloatRect targetBoundingBox = object.objectBoundingBox();
 
     // Resource was not layouted yet. Give back the boundingBox of the object.
     if (selfNeedsLayout())
         return targetBoundingBox;
 
-    FloatRect clipRect = object.strokeBoundingBox();
-
-    if (layer()->isTransformed())
-        clipRect = layer()->currentTransform(RenderStyle::individualTransformOperations()).mapRect(clipRect);
+    if (m_clipBoundaries[repaintRectCalculation].isEmpty())
```

Comment:
Oh, the caching logic no longer works for LBSE, after my resource invalidation patch, you cannot rely on removeAllClientsFrom...

I would not cache m_clipBoundaries at all, just compute it.

Reply:
Done.